db/migrate/20151007014640_create_heating_element_statuses.rb

```diff
@@ -0,0 +1,13 @@
+class CreateHeatingElementStatuses < ActiveRecord::Migration
+  def change
+    create_table :heating_element_statuses do |t|
+      t.string :device_id
+      t.integer :heating_element_id
```

Comment:
I know I'm kinda flip-flopping on this one, but to be consistent with everything else I think this needs to be ```:heat_id ```